packages/rum-core/src/domain/rumEventsCollection/action/listenEvents.ts

```diff
@@ -0,0 +1,53 @@
+import { addEventListener, DOM_EVENT } from '@datadog/browser-core'
+
+export type OnClickCallback = (clickEvent: MouseEvent & { target: Element }, hasSelectionChanged: boolean) => void
+
+export function listenEvents({ onClick }: { onClick: OnClickCallback }) {
```

Comment:
Mmh I changed the name because it doesn't listen to click events only now. Also in the future we'll have more callbacks than just clicks. But I can change the name back to `listenClickEvents` in the meantime if you want.

Reply:
Seems fine to me